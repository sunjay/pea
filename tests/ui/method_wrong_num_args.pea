fn main() {
    1.min(2, 3);
}
