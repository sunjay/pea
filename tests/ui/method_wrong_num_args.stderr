[tests/ui/method_wrong_num_args.pea:2:5] error: wrong number of arguments: expected 2, found 3
[tests/ui/method_wrong_num_args.pea:2:7-2:9] note: function being called has signature `fn (i64, i64) -> i64`

error: aborting due to 1 previous error

